pipeline.yml

```diff
@@ -0,0 +1,26 @@
+trigger:
+- develop
+
+variables:
+  SONAR_TOKEN: '5c770b1887ddf62b505d292a8b9d29a94c98f202'
+
+pool:
+  vmImage: 'ubuntu-18.04'
+
+steps:
+- task: SonarCloudPrepare@1
+  inputs:
+    SonarCloud: 'sonar'
+    organization: 'pufke'
+    scannerMode: 'Other'
+- task: Maven@3
+  inputs:
+    mavenPomFile: 'backend-spring-boot/pom.xml'
+    mavenOptions: '-Xmx3072m'
+    javaHomeOption: 'JDKVersion'
+    jdkVersionOption: '1.11'
+    jdkArchitectureOption: 'x64'
+    publishJUnitResults: true
+    goals: 'clean compile'
```

Comment:
Is it enough to do only clean and compile?

Reply:
For now, yes. It will be test or verify here eventually, but the project with the tests isn't set up correctly yet (it tries to use the real database cause H2 isn't set up) and we have no tests.

Reply:
Ok, nice!